crates/fuel-core/src/graphql_api/storage.rs

```diff
@@ -0,0 +1,54 @@
+use fuel_core_storage::kv_store::StorageColumn;
+
+pub mod receipts;
+pub mod transactions;
+
+/// GraphQL database tables column ids to the corresponding [`fuel_core_storage::Mappable`] table.
+#[repr(u32)]
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    strum_macros::EnumCount,
+    strum_macros::IntoStaticStr,
+    PartialEq,
+    Eq,
+    enum_iterator::Sequence,
+    Hash,
+)]
+pub enum Column {
+    /// The column id of metadata about the blockchain
+    Metadata = 0,
+    /// See [`Receipts`](receipts::Receipts)
+    Receipts = 1,
+    /// The column of the table that stores `true` if `owner` owns `Coin` with `coin_id`
+    OwnedCoins = 2,
+    /// Transaction id to current status
+    TransactionStatus = 3,
+    /// The column of the table of all `owner`'s transactions
+    TransactionsByOwnerBlockIdx = 4,
+    /// The column of the table that stores `true` if `owner` owns `Message` with `message_id`
+    OwnedMessageIds = 5,
```

Comment:
What is the difference between a "Message" and an "Event". I remember in the last PR you changed a lot of "message"s to "event"s.

Reply:
The `Message` is something spendable on the Fuel blockchain, while the `Event` is something bridged from the L1. `Event::Message` and `Message` are basically the same, but we can have more events from L1(like forced transactions).